<!--! Template for security announcement -->
<html xmlns:py="http://genshi.edgewall.org/" xmlns:xi="http://www.w3.org/2001/XInclude" py:strip="">


<py:def function="announcement_id">
PMASA-2011-8
</py:def>

<py:def function="announcement_date">
2011-07-xx
</py:def>

<py:def function="announcement_summary">
Possible directory traversal.
</py:def>

<py:def function="announcement_description">
Fixed filtering of a file path in the MIME-type transformation code, which allowed for directory traversal.
</py:def>

<py:def function="announcement_severity">
We consider this vulnerability to be serious.
</py:def>

<py:def function="announcement_affected">
The 3.4.3 and earlier versions are affected.
</py:def>

<py:def function="announcement_solution">
Upgrade to phpMyAdmin 3.3.10.2 or 3.4.3.1 or apply the related patch listed below.
</py:def>

<!--! Links to reporter etc, do not forget to escape & to &amp; -->
<py:def function="announcement_references">
This issue was found by Frans Pehrson from <a href="http://www.xxor.se">Xxor AB</a>
</py:def>

<!--! CVE ID of the report, this is automatically added to references -->
<py:def function="announcement_cve">CVE-2011-2508</py:def>

<py:def function="announcement_cwe">661</py:def>

<py:def function="announcement_commits">
5ee357a572866e730d83f56d6187a67c7c48e523
b434320eff8ca9c2fc1b043c1804f868341af9a7
</py:def>

<xi:include href="_page.tpl" />
</html>
